mono/utils/racing.h

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
You need to add this file to `mono/utils/Makefile.am` and `msvc/libmonoutils.vcxproj`, `msvc/libmonoutils.vcxproj.filters`.